### DISCUSSION TITLE
Severity Assumptions

### DISCUSSION THREAD

**Discussion by adenooy** [neherlab/covid19_scenarios#745] [General] [answered]
Hi there,

This has been a really fantastic tool and I've enjoyed exploring it.

I wanted to check, I've looked at creating a JSON file to upload for the scenario parameters and I think it might have worked well previously. What I'm struggling with at the moment is the age specific severity parameters. At this point the age distribution population numbers are updating, but it doesn't look like any of the actual severity values are changing.

I'm not sure if perhaps I've created the file wrong. I did also try download an example one, change the assumptions and re-load it, but the changes did not seem to be uploaded.

I was wondering if there might be any advice on a mistake I've perhaps made or how I might be able to move forward without manually typing in these assumptions each time?

Thanks again for the fantastic tool!

**Comment by ivan-aksamentov** [comment, marked as answer]
Hi @adenooy , thanks for the report. This sounds like a bug, I will try to reproduce it.
Could you please share an example of a file you've been using?

**Reply by ivan-aksamentov** [reply]
@adenooy I was able to reproduce and [fix](https://github.com/neherlab/covid19_scenarios/commit/14d6a395ba9f36cf06d2dafd45b06db1178dbe00) the issue. It will be released in version 1.6.5 in a few minutes. Please verify that you are using version 1.6.5 (you can see the currently used version in the left part of the footer in the bottom of the page) and let me know if it fixes it for you. Thanks again for taking time to report!

**Reply by adenooy** [reply]
Thank you so much! Seems like its updating now, really appreciate all the help.

**Reply by tiagoandresvaz** [reply]
Thanks @ivan-aksamentov  and @adenooy
